### DISCUSSION TITLE
Database Size Exceeds Free Limit Incorrectly + Large Unused Items

### DISCUSSION THREAD

**Discussion by meetyan** [supabase/supabase#30631] [Questions]
Hello,

A few days ago, I received an email from the Supabase team stating that my database size had exceeded the free limit. However, after checking and reviewing the backend, I found that it hasn’t exceeded the limit—all my tables combined are less than 100MB. 

<img width="801" alt="Screenshot 2024-11-24 at 10 59 02" src="https://github.com/user-attachments/assets/e33f4369-1107-4d79-bf07-e001c761f486">

Additionally, after receiving the email, I deleted the largest table in terms of storage usage, and the total size is now well within the 500MB free tier limit.

While inspecting the database schema, I noticed two items with unusually large storage usage: net._http_response (430MB) and supabase_functions.hooks (111MB). I don’t recall using either of these. Could anyone explain why they are taking up so much space and advise how I can clean them up?

<img width="1034" alt="Screenshot 2024-11-24 at 10 58 29" src="https://github.com/user-attachments/assets/74676e76-02cd-4bb9-901c-a722b31ecc8e">

<img width="1025" alt="Screenshot 2024-11-24 at 10 58 38" src="https://github.com/user-attachments/assets/383fd0cd-4a43-4695-85b0-eb53a3d38a96">

**Comment by GaryAustin1** [comment]
net.http_response is pg_net (used by webhooks) but I thought it cleaned up itself.

hooks is also associated with webhooks also.  But it is supposed to be the webhooks you have created.  So 1 entry per webhook.  That seems very odd and sort of agrees with your net table that you have created way too many webhooks somehow.
